point1:
    position:
      x: 0.0751908328797879
      y: -0.005938880116238443
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: 0.2578558786421914
      w: 0.9661833914167969

point2:
    position:
      x: 4.992138026379149
      y: -2.1813975957392184
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: -0.647473102521833
      w: 0.7620883029615085